service-api/module/Application/src/Yoti/SessionStatusService.php

```diff
@@ -77,8 +81,41 @@ private function handleSessionCompletion(CaseData $caseData): ?CounterService
         return $caseData->counterService;
     }
 
-    private function evaluateFinalResult(array $checks): bool
+    private function getDocumentScanned(string $mediaId, string $yotiSessionId): array
+    {
+        $nonce = strval(Uuid::uuid4());
+        $timestamp = (new DateTime())->getTimestamp();
+        $result = [];
+
+        try {
+            $result = $this->yotiService->retrieveMedia($yotiSessionId, $mediaId, $nonce, $timestamp);
+        } catch (YotiException $e) {
+            $this->logger->error('Yoti media result error: ' . $e->getMessage());
+        } catch (InvalidArgumentException $exception) {
+            $this->logger->error('Error retreiving media results: ' . $exception->getMessage());
+        }
+
+        return $result["response"];
+    }
+
+    private function evaluateFinalResult(array $checks, ?string $mediaId, CaseData $caseData): bool
     {
+        //If UK passport ensure document presented was in date range
+        if (is_string($mediaId) && $caseData->idMethod === "po_ukp") {
+            $documentScanned = $this->getDocumentScanned($mediaId, $caseData->yotiSessionId);
+
+            if (is_string($documentScanned["expiration_date"])) {
+                $expiry = new DateTime($documentScanned["expiration_date"]);
+
+                $currentDate = new DateTime();
+                $acceptDate = (clone $currentDate)->modify('-18 months');
```

Comment:
Can save yourself a clone here:
```suggestion
                $acceptDate = (new DateTime())->modify('-18 months');
```